data/governments.txt

```diff
@@ -403,6 +403,39 @@ government "Heliarch"
 		"Quarg" -.01
 		"Coalition" 1
 
+government "Hicemus"
+	swizzle 0
+	color 0.6 0.0 0.4
+	"crew attack" .2
+	"crew defense" 1
+	language "Incups"
+	"player reputation" 0
+	"attitude toward"
+		"Incipias" 0.1
+		"Incipias Civilian" 1
+
+government "Incipias"
```

Comment:
These are the non civilians then? The military?

Reply:
Basically yes, I am still searching for a name for the old government to distinguish them better